cli.go

```diff
@@ -38,15 +39,26 @@ func NewCli() *cobra.Command {
 
 			Vidx := NewVidx()
 			Pidx := NewPidx()
+			Pidx.SetForce(flags.force)
 
+			CacheDir = flags.cacheDir
+
+			ExitOnError(EnsureDir(CacheDir))
 			ExitOnError(Vidx.Init(vidxFileName))
-			ExitOnError(Pidx.Update(Vidx))
+
+			err := Pidx.Update(Vidx)
+			if err != nil {
+				Logger.Error(err.Error())
```

Comment:
For now it's OK, but I would consider using Go's `log` package or maybe `logrus`. A logging package is not something you will want to maintain over time and these other packages can add a lot to debugging.

Reply:
Thanks for the tips, I'll check into moving logs to `logrus` later this week